tests/test-distribution/test-distribution-common/pom.xml

```diff
@@ -245,6 +245,16 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <systemPropertyVariables combine.children="append">
```

Comment:
How were the tests retrieving the system properties for the infinispan docker versions before?

Reply:
> How were the tests retrieving the system properties for the infinispan docker versions before?

they were not but just using the hardcoded version. This possible dynamic feature has been lost long time ago when moving modules around for jetty-12 new sources tree.